packages/functions-utils/tests/main.js

```diff
@@ -0,0 +1,81 @@
+const test = require('ava')
+const pathExists = require('path-exists')
+const del = require('del')
+const { tmpName, dir: tmpDir } = require('tmp-promise')
+
+const { add } = require('..')
+
+const FIXTURES_DIR = `${__dirname}/fixtures`
+
+const PREFIX = 'test-functions-utils'
+
+test('Single file', async t => {
```

Comment:
How about `should copy function file to dist directory` ?
Same comment for other tests (e.g. `should throw when source is undefined`

Reply:
Good point! Just improved the test titles accordingly.